ext/agent/jobs/error_discovery.go

```diff
@@ -13,7 +14,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func ScanForErrors(cli *client.Client, logger *logrus.Logger, bearerToken string) {
+func ScanForErrors(cli *client.Client, logger *logrus.Logger, taskPayload models.TaskPayload) {
```

Comment:
I think we can just call it `client` instead of `cli` as it will be easier to understand

Reply:
changed to dockerClient bacuse of library name conflict wiht "client"